pkg/backup/backup/backup_cleaner.go

```diff
@@ -55,11 +57,23 @@ func (bc *backupCleaner) Clean(backup *v1alpha1.Backup) error {
 	}
 	ns := backup.GetNamespace()
 	name := backup.GetName()
+	backupJobName := backup.GetBackupJobName()
+
+	klog.Infof("start to ensure that backup %s/%s job %s have finished", ns, name, backupJobName)
+
+	finished, err := bc.ensureBackupJobFinished(backup)
+	if err != nil {
+		return fmt.Errorf("ensure %s/%s job %s finished failed: %s", ns, name, backupJobName, err)
+	}
+	if !finished {
```

Comment:
caller of `Clean` no need to know if it cleans success or not finished?

Reply:
`Clean` just deploy a job. The result of clean is represented by the `Backup`'s status